<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterAlertWidget</class>
 <widget class="QWidget" name="FilterAlertWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>182</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">#wFilterAlertContainer
{
border: 1px solid;
border-color: #FFD8D9DB; /*colorToken.border-disabled*/
border-radius: 8px;
}
QPushButton
{
border: none;
background:transparent;
}
QPushButton.menu-style
{
color: #FF303233;/*colorToken.text-primary*/
font-size: 14px;
line-height: 20px;
text-align: left;
}
QWidget.menu-style:hover
{
background-color:#FFF7F7F7;/*colorToken.surface-1*/
}
QWidget.menu-style:pressed
{
background-color:#FFEFEFF0;/*colorToken.surface-2*/
}
QWidget.menu-style[active=&quot;true&quot;]
{
background-color:#FFEFEFF0;/*colorToken.surface-2*/
}
#wAllNotifications, #wContactsNotifications, #wPaymentNotifications, #wSharesNotifications
{
border-radius: 6px;
}
#bNumberNotifications, #bNumberNotificationsContacts, #bNumberNotificationsPayments, #bNumberNotificationsShares
{
border: none;
background-color: #FFDD1405;/*colorToken.icon-brand*/
color: #FFFAFAFB;/*colorToken.text-inverse-accent*/
font-weight: 600;
font-size: 8px;
line-height: 12px;
border-radius:7px;
margin-right: 8px;
}

#wSeparator
{
background-color: #FFEFEFF0;/*colorToken.surface-2*/
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="wFilterAlertContainer" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QWidget" name="wAllNotifications" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="class" stdset="0">
         <string notr="true">menu-style</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0">
         <property name="spacing">
          <number>12</number>
         </property>
         <property name="leftMargin">
          <number>16</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="bAllIcon">
           <property name="minimumSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="icon">
            <iconset resource="../Resources_light.qrc">
             <normaloff>:/bell.svg</normaloff>:/bell.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="autoDefault">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bAll">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>All notifications</string>
           </property>
           <property name="class" stdset="0">
            <string notr="true">menu-style</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="bNumberNotifications">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>14</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>14</height>
            </size>
           </property>
           <property name="text">
            <string notr="true">0</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="wSeparator" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>1</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>1</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="wTypesNotifications" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="wContactsNotifications" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="class" stdset="0">
            <string notr="true">menu-style</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
            <property name="spacing">
             <number>12</number>
            </property>
            <property name="leftMargin">
             <number>16</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="bContactsIcon">
              <property name="minimumSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="icon">
               <iconset resource="../Resources_light.qrc">
                <normaloff>:/users.svg</normaloff>:/users.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="bContacts">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Contacts</string>
              </property>
              <property name="class" stdset="0">
               <string notr="true">menu-style</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="bNumberNotificationsContacts">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>24</width>
                <height>14</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>14</height>
               </size>
              </property>
              <property name="text">
               <string notr="true">0</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="wSharesNotifications" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="class" stdset="0">
            <string notr="true">menu-style</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0,0">
            <property name="spacing">
             <number>12</number>
            </property>
            <property name="leftMargin">
             <number>16</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="bSharesIcon">
              <property name="minimumSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="icon">
               <iconset resource="../Resources_light.qrc">
                <normaloff>:/folder-users.svg</normaloff>:/folder-users.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="autoDefault">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="bShares">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Incoming Shares</string>
              </property>
              <property name="class" stdset="0">
               <string notr="true">menu-style</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="bNumberNotificationsShares">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>24</width>
                <height>14</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>14</height>
               </size>
              </property>
              <property name="text">
               <string notr="true">0</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="wPaymentNotifications" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="class" stdset="0">
            <string notr="true">menu-style</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,0,0">
            <property name="spacing">
             <number>12</number>
            </property>
            <property name="leftMargin">
             <number>16</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="bPaymentIcon">
              <property name="minimumSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="icon">
               <iconset resource="../Resources_light.qrc">
                <normaloff>:/currency-dollar.svg</normaloff>:/currency-dollar.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="autoDefault">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="bPayment">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Payment</string>
              </property>
              <property name="class" stdset="0">
               <string notr="true">menu-style</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="bNumberNotificationsPayments">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>24</width>
                <height>14</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>14</height>
               </size>
              </property>
              <property name="text">
               <string notr="true">0</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resources_light.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>bAllIcon</sender>
   <signal>clicked()</signal>
   <receiver>bAll</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>60</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bNumberNotifications</sender>
   <signal>clicked()</signal>
   <receiver>bAll</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
